modules/logging-release-notes-5-6-27.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+// logging-5-6-release-notes
+:_mod-docs-content-type: REFERENCE
+[id="logging-release-notes-5-6-27_{context}"]
+= Logging 5.6.27
+This release includes link:https://access.redhat.com/errata/RHBA-2024:10988[RHBA-2024:10988].
+
+[id="logging-release-notes-5-6-27-bug-fixes_{context}"]
+== Bug fixes
+None.
+
+[id="logging-release-notes-5-6-27-CVEs_{context}"]
+== CVEs
+
+* link:https://access.redhat.com/security/cve/CVE-2018-12699[CVE-2018-12699]
+* link:https://access.redhat.com/security/cve/CVE-2019-12900[CVE-2019-12900]
+* link:https://access.redhat.com/security/cve/CVE-2024-9287[CVE-2024-9287]
+* link:https://access.redhat.com/security/cve/CVE-2024-10041[CVE-2024-10041]
+* link:https://access.redhat.com/security/cve/CVE-2024-10963[CVE-2024-10963]
+* link:https://access.redhat.com/security/cve/CVE-2024-11168[CVE-2024-11168]
+* link:https://access.redhat.com/security/cve/CVE-2024-35195[CVE-2024-35195]
```

Comment:
@anpingli I Removed the following:
* link:https://access.redhat.com/security/cve/CVE-2024-27043[CVE-2024-27043]
* link:https://access.redhat.com/security/cve/CVE-2024-27399[CVE-2024-27399]
* link:https://access.redhat.com/security/cve/CVE-2024-38564[CVE-2024-38564]
* link:https://access.redhat.com/security/cve/CVE-2024-46858[CVE-2024-46858]

And added * link:https://access.redhat.com/security/cve/CVE-2024-35195[CVE-2024-35195]

Reply:
LGTM